application-templates/javascript/event/.nvmrc

```diff
@@ -1 +1 @@
-16.16.0
\ No newline at end of file
+v20.15.1
```

Comment:
@ruidias-commercetools can we update to the latest LTS version please?

Reply:
And if possible, can we also ensure the same version in the package.json: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines